Documentation/contributing/development/images.rst

```diff
@@ -205,12 +205,13 @@ Update cilium-builder and cilium-runtime images
 
    .. code-block:: shell-session
 
-       $ git commit -s images/ -m "update cilium-{runtime,builder}"
+       $ git commit -s -m "update cilium-{runtime,builder}"
```

Comment:
I think we either need to specify the paths to be committed or the '-a' option. Otherwise nothing will be committed. Unless we do 'git add' the changes before, but the docs currently don't mention that. Same for the 'git commit' commands below.

Reply:
You are absolutely right, I normally do `git add` to select files before commit but I think `git commt -a` works well in this context, added. PTAL.